frontend/src/lib/dashbaordVariables/getDashboardVariables.ts

```diff
@@ -7,24 +7,28 @@ export const getDashboardVariables = (): Record<string, unknown> => {
 			globalTime,
 			dashboards: { dashboards },
 		} = store.getState();
-		const [selectedDashboard] = dashboards;
-		const {
-			data: { variables = {} },
-		} = selectedDashboard;
 
-		const { start, end } = getStartEndRangeTime({
-			type: 'GLOBAL_TIME',
-			interval: globalTime.selectedTime,
-		});
+		if (dashboards.length > 0) {
```

Comment:
this change is not relevant please remove this change

Reply:
Done